core/common/src/TextureMapping.ts

```diff
@@ -96,6 +98,31 @@ export namespace TextureMapping { // eslint-disable-line no-redeclare
     public static readonly identity = new Trans2x3();
   }
 
+  /** Properties used to construct a [[TextureMapping.ConstantLodParams]]. */
```

Comment:
Should we add a NextVersion.md entry?

Reply:
I don't know.  Do we need to wait until backend stuff is implemented for it (I'm not even sure what that might entail).

Reply:
Makes sense to me @MarcNeely. Hopefully that will end up in the 4.0 release anyway.

Reply:
Sounds good.